crates/sargon/src/profile/mfa/security_structures/roles/factor_levels/factor_instance_level/role_with_factor_instances.rs

```diff
@@ -0,0 +1,83 @@
+use crate::prelude::*;
+
+pub(crate) type RoleWithFactorInstances<const ROLE: u8> =
+    AbstractBuiltRoleWithFactor<ROLE, FactorInstance>;
+
+impl<const ROLE: u8> RoleWithFactorSources<ROLE> {
```

Comment:
this should probably be in `role_with_factor_sources.rs`

Reply:
FIXED